pkg/kubernetes/namespace.go

```diff
@@ -90,13 +89,13 @@ func (ns *namespaceRepository) WatchPhase(emit resource.EventEmitter) error {
 	watcher, err := ns.kubernetes.CoreV1().Namespaces().Watch(metav1.ListOptions{})
 
 	if err != nil {
-		log.Printf("[WATCHER] %s", err.Error())
+		logrus.Errorf("Error when watching phase : %s", err.Error())
 		return err
 	}
 
 	defer func() {
 		watcher.Stop()
-		log.Printf("[WATCHER] restart watcher due to connection close")
+		logrus.Info("restarting pahse watcher due to connection close")
```

Comment:
typo restarting ***pahse*** watcher due to connection close

Reply:
deleted logs due to refactoring on previous branch.